gulpfile.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+/* eslint no-sync: 0*/
+const gulp = require('gulp');
+const rename = require('gulp-rename');
+
+gulp.task('flow', function () {
+  return flowType('src', 'lib');
+});
+
+// ........functions .......
+function flowType(src, dst) {
+  const srcPath = [ src + '/**/*.js',
+    '!' + src + '/**/__tests__/**', '!' + src + '/**/__mocks__/**' ];
+  return gulp
+    .src(srcPath)
+    .pipe(rename({extname: '.js.flow'}))
+    .pipe(gulp.dest(dst));
```

Comment:
`Graphql` and `Graphql-relay` are suitable for output flow type by `.js.flow`. Because they do not use `@providesModule`,just need to copy and rename `.js` to `.js.flow`. Flow will work, and do not break anything in converted code . 
I tried fbjs's covert plugin for `@providesModule`(which used by React&Relay),seems the plugin have some bugs to process some syntax.